src/plugins/Network.js

```diff
@@ -47,26 +56,188 @@ module.exports = class Network extends OverlayPlugin {
+        mobileGroup.appendChild(this.mobileStatus);
+        this.mobileGroupSection.appendChild(mobileGroup);
+
+        // Generate input rows for network mobile profiles
+        const networkTypeInputLabel = document.createElement('div');
+        networkTypeInputLabel.className = 'input_label';
+        networkTypeInputLabel.innerHTML = 'Network type:';
+
+        this.profileInputs.appendChild(networkTypeInputLabel);
+        this.selectMobileProfile = document.createElement('select');
+        this.selectMobileProfile.onchange = this.changeMobileProfile.bind(this);
+        // Add option for each child
+        MOBILE_PROFILES.slice().reverse()
+            .forEach((profile) => {
+                // 5g is available only for version >= 10
+                if (profile.name === '5g') {
+                    return;
+                }
+                const option = new Option(profile.label, profile.name);
+                this.selectMobileProfile.add(option);
+            });
+        this.profileInputs.appendChild(this.selectMobileProfile);
+
+        // Generate input rows for signal strength
+        const signalStrengthInputLabel = document.createElement('div');
+        signalStrengthInputLabel.className = 'input_label';
+        signalStrengthInputLabel.innerHTML = 'Signal strength:';
+        this.inputMobileSignalStrength.appendChild(signalStrengthInputLabel);
+
+        this.selectMobileSignalStrength = document.createElement('select');
+        this.selectMobileSignalStrength.onchange = this.changeMobileSignalStrength.bind(this);
+        this.inputMobileSignalStrength.appendChild(this.selectMobileSignalStrength);
+        // Add option for each child
+        MOBILE_SIGNAL_STRENGTH.slice().reverse()
+            .forEach((strength) => {
+                const option = new Option(strength.label, strength.name);
+                this.selectMobileSignalStrength.add(option);
+            });
+
+        this.updateMobileSectionStatus();
+    }
+
+    disableMobileThrottling() {
+        this.mobilethrottling = false;
+
+        // Generate input rows for network profiles
+        this.selectProfile = document.createElement('select');
+        const defaultOption = new Option(this.i18n.NETWORK_DELECT_PROFILE || 'Select a profile');
+        this.selectProfile.add(defaultOption);
+        this.selectProfile.onchange = this.changeProfile.bind(this);
+        // Add option for each child
+        PROFILES.slice().reverse()
+            .forEach((profile) => {
+                const option = new Option(profile.label, profile.name);
+                this.selectProfile.add(option);
+            });
+        this.profileInputs.appendChild(this.selectProfile);
+    }
+
+    enable5G() {
+        const profile = MOBILE_PROFILES.at(0);
+        const option = new Option(profile.label, profile.name);
+        this.selectMobileProfile.add(option);
+    }
+
+    disable5G() {
+        // Nothing to do!
```

Comment:
Why?

Reply:
because the widget is constructed by default without 5G
if capability is true we add it if its fals we have nothing to do